replay/models/kl_ucb.py

```diff
@@ -0,0 +1,156 @@
+import math
+
+from typing import Optional
+from replay.models import UCB
+from pyspark.sql.types import DoubleType
```

Comment:
should be checked for pyspark availability, if PYSPARK_AVAILABLE...
Please see example here: https://github.com/sb-ai-lab/RePlay/blob/main/replay/models/knn.py